packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.service.js

```diff
@@ -32,18 +55,21 @@ export default class {
   }
 
   getServiceTypes() {
-    return this.getServiceTypesRaw().then((serviceTypes) =>
-      [...serviceTypes]
-        .map((serviceType) => ({
-          ...serviceType,
-          label: this.$translate.instant(
-            `ovhManagerSupport_new_serviceType_${serviceType.name}`,
+    return this.getServiceTypesRaw()
+      .then((serviceTypes) => this.constructor.addSubTypes(serviceTypes))
+      .then((serviceTypes) =>
+        [...serviceTypes]
```

Comment:
You can directly use the array (instead of spread into new Array]

Reply:
This is done. Thank you